bait-files/src/main/java/com/google/sps/servlets/ReviewServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.googlecode.objectify.ObjectifyService;
+import java.io.IOException;
+import java.lang.String;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/review")
+public class ReviewServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // TODO: Add error checking and redirection for non-Numeric/non-long requestId values
+    long requestId = Long.parseLong(request.getParameter("requestId"));
+    Request userRequest = ObjectifyService.ofy().load().type(Request.class).id(requestId).now();
```

Comment:
So exciting that you got Objectify working!  =D

One thing I don't quite understand... in this PR you have created both a Request and a Review class... and it looks like you are only using one of them at this point (as of this PR).  And you've implemented the /review GET endpoint to return a Request.. rather than a Review?  Shouldn't it be retrieving a Review object instead?

Reply:
Or I guess, looking at the rest of the code... if your intention is to have the reviewing agent retrieve the Request from the server, and to not consider the data to be a "Review" until the agent submits the results, then I would say to rename his file to RequestServlet if you want GET to return a Request.  That way it makes more sense for the frontend to ask the backend for "/request"... which then returns a Request =)

Reply:
So the intention is to have a url like "/review.html?requestId=123456" retrieve the Request object and add the data to the review form. The doPost() for this file is going to save the results of this form as a Review object.